templates/test.template

```diff
@@ -2,21 +2,20 @@
 // This code is governed by the BSD license found in the LICENSE file.
 
 /*---
-template: default
 author: Mathias Bynens
-desc: Unicode property escapes `\p{<%= UnicodePropertyValueExpression %>}` and `\P{<%= UnicodePropertyValueExpression %>}`
+description: >
+  Unicode property escapes `\p{}` and \P{}` for `<%= UnicodePropertyValueExpression %>`
 esid: sec-static-semantics-unicodematchproperty-p
-flags: [generated]
 features: [regexp-unicode-property-escapes]
 ---*/
 
 assert(
 	/^\p{<%= UnicodePropertyValueExpression %>}+$/u.test(<%= matchSymbols %>),
-	"\\p{<%= UnicodePropertyValueExpression %>} doesn't match all symbols it should match"
+	"\\p{<%= UnicodePropertyValueExpression %>} matches all proper symbols"
```

Comment:
in test262 we conventionally use positive messages. We might also use ` x should match all proper symbols`  if it reads better.